JeffShepherd/Rancid-Tomatillos
# Feature/search and filter

## What does this PR do?
- Applies sorting functionality to the homepage

## How should it be tested in the UI?
- cd into Rancid-Tomatillos repo, run `npm start`
- Select a dropdown value next to the "Sort by: " label
- On change, the cards on the homepage should be reorganized in descending order
- Choosing a second value should reorganize the cards based on the corresponding property
- Filtered search results should still be sortable
- Clicking the home button should return the order of the cards to their original state
